src/labels/CustomLabels.labels

```diff
@@ -2025,6 +2025,22 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Error when the component is used on a Non-Recurring Donations page</shortDescription>
         <value>You must enable Enhanced Recurring Donations to use this component.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ScheduleVisualizerFLSInstallment</fullName>
```

Comment:
@sam-knox , can you please review messages displayed when user does not have field-level security on fields related to information displayed on Active Schedules and Upcoming Installments?